tasks/certbot_nginx.yml

```diff
@@ -112,97 +112,41 @@
 
 # 5.
 
+- name: create dirs (locally under tmp, for templating)
+  file:
+    path: "{{ item }}"
+    state: directory
+  with_items:
+    - "{{ nginx_localhost_config_dir }}"
+    - "{{ nginx_localhost_config_dir }}/http"
+    - "{{ nginx_localhost_config_dir }}/stream"
+  delegate_to: localhost
+  become: false
+
+- name: template nginx config files (locally)
+  template:
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+  with_items:
+    - { src: nginx.conf.j2, dest: "{{ nginx_localhost_config_dir }}/nginx.conf" }
+    - { src: nginx-default.conf.j2, dest: "{{ nginx_localhost_config_dir }}/http/default.conf" }
+    - { src: nginx-stream.conf.j2, dest: "{{ nginx_localhost_config_dir }}/stream/restund.conf" }
+  delegate_to: localhost
+  become: false
```

Comment:
The nginx role supports only uploading files, not templates. Since I need to template files, I do the templating first, and then upload the templated files.

Reply:
Okay, understood. Then, I have to ask, What is the role is still needed for?